products/cloudfunctions/terraform.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+legacy_name: 'cloudfunctions'
+overrides: !ruby/object:Overrides::ResourceOverrides
+  CloudFunction: !ruby/object:Overrides::Terraform::ResourceOverride
+    legacy_name: 'google_cloudfunctions_function'
+    id_format: '{{project}}/{{region}}/{{cloud_function}}'
+    base_url: projects/{{project}}/locations/{{region}}/functions
+    import_format: ["projects/{{project}}/locations/{{region}}/functions/{{cloud_function}}"]
+    exclude_resource: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloudfunctions_cloud_function"
+        skip_test: true
```

Comment:
Do we not want a generated IAM test for this because it requires setting up the zip?

Reply:
(if that's the case, see the SSL cert examples. There's a magic string that gets replaced by values in `test-fixtures`)

Reply:
will do riley's suggestion